src/juicy-popover.html

```diff
@@ -209,93 +103,201 @@
+                var expandable = this.shadowRoot.querySelector("#expandable");
+                
+                handle.addEventListener("mousedown", function (ev) {
+                    if (!expandable.classList.contains("expanded")) {
+                        this.expand();
+                        self.ignore = true;
+                    }
+                }.bind(this));
 
-        function doesItFit(bounding) {
-            //if this fails then switch to the library https://github.com/patik/within-viewport
-            return (
-                    bounding.top >= 0 &&
-                    bounding.left >= 0 &&
-                    bounding.bottom <= document.body.clientHeight &&
-                    bounding.right <= document.body.clientWidth
-                );
-        }
+                handle.addEventListener("mouseup", function (ev) {
+                    if (ev.target.nodeName == "INPUT") {
+                        self.ignore = true; //just a friendly feature - don't collapse the expandable if clicked on a handler which is an input
+                    }
+                }.bind(this));
 
-        JuicyPopoverElementPrototype.reposition = function () {
-            var container = this.shadowRoot.querySelector("#container");
-            var expandable = this.shadowRoot.querySelector("#expandable");
-            var handle = this.shadowRoot.querySelector("#handle");
+                expandable.addEventListener("mousedown", function (ev) {
+                    self.ignore = true;
+                }.bind(this));
 
-            var config = this.getAttribute("position") || "bottom left, bottom beforeright, beforetop left, beforetop beforeright";
-            var positions = config.split(",");
-            var hasFallbacks = (positions.length > 1);
+                window.addEventListener("mouseup", function (ev) {
+                    if (!self.ignore && expandable.classList.contains("expanded")) {
+                        this.collapse();
+                    }
+                    self.ignore = false;
+                }.bind(this));
 
-            var expandable = this.shadowRoot.querySelector("#expandable");
-            if (expandable.classList.contains("expanded")) {
-                this._hideExpandable();
+                window.addEventListener("resize", function (e) {
+                    if (expandable.classList.contains("expanded")) {
+                        this.reposition();
+                    }
+                }.bind(this));
+                // This will work only in V1
+                shadowRoot.getElementById('deprecated-v1').addEventListener('slotchange', function(ev){
+                    const slottedElements = ev.target.assignedNodes();
+                    console.warn('Slot name `expendable` is deprecated, please use `expandable`!', slottedElements);
+                });
+                
+                return self;
             }
+            
+            attributeChangedCallback(attr, oldVal, newVal) {
```

Comment:
No attributes are observed, so this callback will never be triggered.
You need to set `static get observedAttributes() {return ['disabled', 'expanded']; }`